src/github/graphql.ts

```diff
@@ -1,3 +1,5 @@
+import { IAccount } from "./interface";
+
```

Comment:
I guess it's from auto import and it misplaces the code ;)

Reply:
Thanks, fixed.